lib/options.json

```diff
@@ -389,7 +389,7 @@
       "bonjour": "should be {Boolean} (https://webpack.js.org/configuration/dev-server/#devserverbonjour)",
       "ca": "should be {String|Buffer}",
       "cert": "should be {String|Buffer}",
-      "clientLogLevel": "should be {String} and equal to one of the allowed values\n\n [ 'info', 'warn', 'error', 'debug', 'trace', 'silent' ]\n\n (https://webpack.js.org/configuration/dev-server/#devserverclientloglevel)",
+      "clientLogLevel": "should be {String} and equal to one of the allowed values\n\n [ 'none', 'info', 'error', 'warning' ]\n\n (https://webpack.js.org/configuration/dev-server/#devserverclientloglevel)",
```

Comment:
The description should have `warn` and `silent`, and we should update documentation.

```diff
- [ 'none', 'info', 'error', 'warning' ]
+ [ 'none', 'silent', 'info', 'error', 'warning', 'warn' ]
```

Reply:
Thank you.
I tried `warn` and `silent`, but it doesn't work as well 🤔 Is it intended?

Reply:
Ah~~ it's my bad. The version of `webpack-dev-server` in my local is old. I'll fix it.
Then can I handle to update documentation?

Reply:
Oops, yep. I confused. `none` and `warning` are going to be deprecated at the next major version, instead of `silent` and `warn`.

Reply:
@sottar We've already had the [updated documentation](https://github.com/webpack/webpack.js.org/pull/2976
). So, I'll merge this documentation after merging this pr.

Reply:
Oh, I see. So the documentation is the latest right? 
It seems different between you suggested above.

Reply:
@sottar sorry, I forgot `trace`...

https://github.com/webpack/webpack-dev-server/blob/master/client-src/default/utils/log.js#L5-L15

please update 🙇